src/main/java/core/basesyntax/DrawInfo.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface DrawInfo {
```

Comment:
The name `DrawInfo` doesn't reflect the behavior it describes. Interfaces should be named after the capability they provide, and in this case, it's drawing something. A more suitable name could be `Drawable` to indicate that implementing classes have the ability to be drawn.